lib/pbio/test/drv/test_bluetooth_btstack.c

```diff
@@ -569,7 +570,18 @@ static const btstack_uart_block_t *test_uart_block_instance(void) {
         .send_block = test_uart_block_send_block,
     };
 
-    return &uart_block;
+    return hci_transport_h4_instance_for_uart(&uart_block);
+}
+
+static const void *test_transport_config(void) {
+    static const hci_transport_config_uart_t config = {
+        .type = HCI_TRANSPORT_CONFIG_UART,
+        .baudrate_init = 1,
+        .baudrate_main = 1,
```

Comment:
I would use realistic numbers here or add a comment explaining why the numbers look funny for testing.